atuin-server/src/settings.rs

```diff
@@ -33,6 +34,7 @@ impl Settings {
             .set_default("host", "127.0.0.1")?
             .set_default("port", 8888)?
             .set_default("open_registration", false)?
+            .set_default("max_history_length", 8192)?
```

Comment:
8192 is the old default set in the database schema, so behaviour does not change